x-pack/plugins/alerting/public/pages/maintenance_windows/types.ts

```diff
@@ -19,23 +24,14 @@ export const RRuleFrequencyMap = {
   '3': RRuleFrequency.DAILY,
 };
 
-export interface RecurringSchedule {
-  freq: RRuleFrequency;
-  interval: number;
-  until?: string;
-  count?: number;
-  byweekday?: string[];
-  bymonthday?: number[];
-  bymonth?: number[];
-}
+export type MaintenanceWindow = Pick<MaintenanceWindowServerSide, 'title' | 'duration' | 'rRule'>;
 
-export type RRule = Partial<RecurringSchedule> & {
-  dtstart: string;
-  tzid: string;
-};
+export type MaintenanceWindowResponse = MaintenanceWindowServerSide &
```

Comment:
nitpick: let's specify that this is a `find` response

Reply:
Resolved in the table pr 7320b3ed2d2cd4f51a0229ec2c5d21712e59abdb